sdk/ml/azure-ai-ml/azure/ai/ml/operations/_registry_operations.py

```diff
@@ -57,7 +62,7 @@ def list(self) -> Iterable[Registry]:
         :rtype: ~azure.core.paging.ItemPaged[Registry]
```

Comment:
How do you pass subscription name per the docstring,  right now the list is defaulted to rg

Reply:
You cannot pass in a subscription as of now. This will be implemented in a follow up PR. I deleted any mention of the unimplemented feature for now.